### PR TITLE
New Port: Griel's Quest for the Sangraal

### PR DESCRIPTION
**Pull request by Cebion**
Game Information
Title: Griel's Quest for the Sangraal
URL:https://github.com/M-HT/griels-quest.git

| Button | Action |
|--|--| 
|DPAD| Move|
|A | Select / Restart Level|
|L1 | Toggle Fullscreen|
|Select| Exit|

CFW Tests:
[X] AmberELEC
[x] ArkOS
[x] MuOS
[x] ROCKNIX
-> [x] Panfrost
-> [x] Adreno
[x] Knulli (Optional)

Resolutions: 
[] 480x320 (Optional)
[X] 640x480
[x] 720x720 (RGB30) (Optional)
[x] Higher resolutions (e.g., 1280

### CODE REVIEW

**Review by kloptops — DISMISSED** [review]

File: ports/grielsquest/Griel's Quest for the Sangraal.sh [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+get_controls
+
+GAMEDIR=/$directory/ports/grielsquest
+CONFDIR="$GAMEDIR/conf"
+BINARY=griels
+
+> "$GAMEDIR/log.txt" && exec > >(tee "$GAMEDIR/log.txt") 2>&1
+
+mkdir -p "$GAMEDIR/conf"
```
Comment (JanTrueno) [inline ports/grielsquest/Griel's Quest for the Sangraal.sh:27]: Confdir is already included with a gitkeep.

File: ports/grielsquest/Griel's Quest for the Sangraal.sh [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+export LD_LIBRARY_PATH="$GAMEDIR/libs.${DEVICE_ARCH}:$LD_LIBRARY_PATH"
```
Comment (JanTrueno) [inline ports/grielsquest/Griel's Quest for the Sangraal.sh:29]: There is no libs folder?

File: ports/grielsquest/Griel's Quest for the Sangraal.sh (hunk 1 above)
Comment (Cebion) [reply ports/grielsquest/Griel's Quest for the Sangraal.sh:27]: > Confdir is already included with a gitkeep.

Indeed, if a user deletes the conf folder by mistake it no longer will work.
So i always create it ^^

File: ports/grielsquest/Griel's Quest for the Sangraal.sh (hunk 2 above)
Comment (Cebion) [reply ports/grielsquest/Griel's Quest for the Sangraal.sh]: > There is no libs folder?

fixed

File: ports/grielsquest/Griel's Quest for the Sangraal.sh (hunk 1 above)
Comment (JanTrueno) [reply ports/grielsquest/Griel's Quest for the Sangraal.sh:27]: Makes sense :)

**Review by JanTrueno — DISMISSED** [review]

**Review by cdeletre — CHANGES_REQUESTED** [review]

**Review by cdeletre — APPROVED** [review]

**Pull request merged**